docs/copilot/language-models.md

```diff
@@ -26,6 +28,21 @@ The list of [models available in Copilot](https://docs.github.com/en/copilot/usi
 > [!NOTE]
 > If you are a Copilot Business or Enterprise user, your administrator needs to enable certain models for your organization by opting in to `Editor Preview Features` in the [Copilot policy settings](https://docs.github.com/en/enterprise-cloud@latest/copilot/managing-copilot/managing-github-copilot-in-your-organization/managing-policies-for-copilot-in-your-organization#enabling-copilot-features-in-your-organization) on GitHub.com.
 
+## Why use your own language model API key?
+
+Developers can now leverage models directly from Anthropic, Azure, Google, OpenAI, OpenRouter, or Ollama directly by providing a valid API key. Learn how to [use your own API in VS Code](#bring-your-own-language-model-key).
+
+Using your own language model API key in VS Code has several advantages:
+
+- **Model choice**: access hundreds of models from different providers, beyond the built-in models.
+- **Experimentation**: experiment with new models or features that are not yet available in the built-in models.
+- **Local compute**: use your own compute for one of the models already supported in GitHub Copilot or to run models not yet available.
+- **Greater control**: by using your own key, you can bypass the standard rate limits and restrictions imposed on the built-in models.
+- **Enhanced observability**: many model providers offer detailed metrics and monitoring capabilities, allowing teams to better track API usage and performance.
```

Comment:
I would remove observability for now.

Enterprises care about observability, and BYOK currently does not work for enterprises.